css/src/components/tag.scss

```diff
@@ -0,0 +1,205 @@
+/* stylelint-disable selector-max-specificity, no-descending-specificity  */
+
+$tag-padding-block: 0.4em !default;
```

Comment:
Not a blocker, but these two values seem like they might have the untended effect of producing award pixel amounts. True? 
16*.4 = 6.4
16*6.5 = 10.4

Would .375em and .675em work instead?